src/devices/bus/epson_qx/multifont.cpp

```diff
@@ -0,0 +1,284 @@
+// license:BSD-3-Clause
+// copyright-holders:Brian Johnson
+/*******************************************************************
+ *
+ * Q10MF Multifont card
+ *   Schematic: https://github.com/brijohn/qx10/raw/master/schematics/multifont/multifont.pdf
+ *   Documentation: http://www.bitsavers.org/pdf/epson/QX-10/options/QX-10-multifont.pdf
+ *
+ *  I/O Ports:
+ *     0xFC:
+ *        Read:
+ *          Reading from this will read a status byte followed by the character data for a requested character.
+ *          Status byte:
+ *          -------------------------------------------------
+ *          | VALID | MODE | 0 0 | UP | RIGHT | LEFT | DOWN |
+ *          -------------------------------------------------
+ *          Valid: 1 = Character is valid, 0 = Character is invalid
+ *          Mode:  1 = Pattern is for CRT, 0 = Pattern is for printer
+ *          Up:    Extends upwards 0 = No, 1 = Yes
+ *          Right: Extends to the right 0 = No, 1 = Yes
+ *          Left:  Extends to the left 0 = No, 1 = Yes
+ *          Down:  Extends downwards 0 = No, 1 = Yes
+ *        Write:
+ *          Data wriiten to this consists of a command byte followed by two bytes representing
+ *          the character code being requested, low byte first. (0x0000-0x0DFF)
```

Comment:
Do you mean “low byte” as in “the byte stored at a lower address” or “least significant byte”?